### PR TITLE
Fix translation of messages with dot at the end

### PR DESCRIPTION
**Pull request by lulco**
If you have some strings like:

Hello.
Hello world.

(with dot at the end), whole message is marked as domain, and id is empty, which leads to translation these messages as empty strings.

### CODE REVIEW

**Comment by lulco** [comment]
@enumag any chance this could be merge to near release?

**Comment by enumag** [comment]
I'm reluctant to fix this. Because then one might expect that any message will work but from what I gather a message with a dot in the middle will split into domain and message anyway, right? Unless this worked before and was broken later then I'd leave it as is I think.

**Comment by lulco** [comment]
Everything you mentioned works:

`Hello.World` works also `Hello world.`

I found that it is partially fixed in extractMessageDomain:
```
if (strpos($message, '.') !== FALSE && strpos($message, ' ') === FALSE) {
```

The only problem is if $id (message) is empty because symfony translator returns empty string:
```
if ('' === $id = (string) $id) {
    return '';
}
```

Maybe I just add condition to extractMessageDomain() that dot should not be last character?

File: src/Translator.php [hunk 1]
```diff
@@ -435,7 +435,7 @@ protected function assertValidLocale($locale)
 	 */
 	private function extractMessageDomain($message)
 	{
-		if (strpos($message, '.') !== FALSE && strpos($message, ' ') === FALSE) {
+		if (strpos($message, '.') !== FALSE && strpos(strrev($message), '.') !== 0 && strpos($message, ' ') === FALSE) {
```
Comment (enumag) [inline src/Translator.php:438]: What do you think about this?

```suggestion
		if (strpos(substr($message, 0, -1), '.') !== FALSE && strpos($message, ' ') === FALSE) {
```

Comment (lulco) [reply src/Translator.php]: it works, I'm in!

**Review by enumag — APPROVED** [review]

**Pull request merged**